modules/examples/src/main/scala/dev/profunktor/redis4cats/RedisScriptsDemo.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2018-2019 ProfunKtor
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
The license is duplicated. It seems it's been commented and then it's been generated automatically again by the plugin.